Styling p.1

bxkr opened the pull request.

bxkr committed:
Floating core, fixed about, font testing

rielyalkov commented on src/app/pages/about/deviz/deviz.component.html:
```diff
@@ -1 +1,9 @@
-<span style="COLOR: #0000cd">Вперёд и вверх! Вперёд и вверх! Вперёд и вверх к победе!</span>
+<pre class="line-division">1) Человек человеку друг!
+2) Никогда не оставляй друга в беде
+3) Работай сам, помогай товарищу
+4) Не жди, когда работа найдёт тебя - ищи её сам
+5) Делай своё дело так, чтобы трое его не переделывали
+6) Знаешь сам - расскажи другому, умеешь сам - научи другого
+7) Не бойся казаться смешным
+8) Лучший подарок маме - это Я!
+</pre>
```
не читаемо, после тега пре с новой строки. Но и в целом тег пре - такое себе, это лучше сделать адаптивным css

rielyalkov commented on src/styles.css:
```diff
@@ -1,5 +1,27 @@
+  width: 100vw;
+}
+
+.fcb {
```
этот класс применяется только в auxiliary-компоненте? зачем он в общих стилях?

rielyalkov replied:
и че за fcb вообще? что это значит? называй переменные нормально

bxkr replied:
крч в общих стилях fcb чтобы можно было сделать картинки на всех страницах, просто указывая (click)="open_floating('url')"

rielyalkov replied:
не понял, а зачем для работы функции нужен css-класс?

rielyalkov commented on src/styles.css:
```diff
@@ -1,5 +1,27 @@
 }
+
+
+#floating {
```
тот же вопрос, что и к fcb

bxkr replied:
тот же ответ, что и к fcb

rielyalkov commented on src/app/pages/gallery/gallery.component.ts:
```diff
@@ -7,7 +7,12 @@ import { Component, OnInit } from '@angular/core';
-  constructor() { }
+  constructor() {}
+
+  open_floating(file) {
```
а нах эта функция в конструкторе?

rielyalkov replied:
если это typeScript, то она еще и void наверно

bxkr replied:
она не в конструкторе

rielyalkov replied:
пардон, проморгал. тогда воид

rielyalkov replied:
А, и еще момент, в названии функции нижние подчеркивания не допустимы. должно быть openFloating()

rielyalkov commented on src/app/pages/gallery/gallery.component.html:
```diff
@@ -1,9 +1,9 @@
-        consectetur adipiscing elit. Fusce porttitor ligula porttitor,
-        lacinia sapien non.</div>
-    </div>
+<div class="hvrbox" (click)='open_floating("https://sun9-21.userapi.com/c850332/v850332545/c6686/guWm0ZskoqM.jpg")'>
```
если уж и разные типы кавычек, то двойные снаружи, одинарные внутри. но в html-шаблоне одинарные юзать вообще не рекомендую

rielyalkov commented on src/app/auxiliary/auxiliary.component.html:
```diff
@@ -1,4 +1,8 @@
 <body>
+  <div id="floating">
+    <mat-icon class="fcb mat-accent" onclick="document.getElementById('floating').style.display = 'none'">close</mat-icon>
```
в angular для таких вещей есть ngClass и ngStyle, удобнее и практичнее

bxkr commented on src/app/auxiliary/auxiliary.component.css:
```diff
@@ -41,12 +41,19 @@ body {
   height: 100vh;
   background: url(../../assets/images/2.jpg) no-repeat center center fixed;
   /*background: #f8f8f8 no-repeat center center fixed;*/
+  overflow-x: scroll;
```
Fix background scrolling bug in some places

bxkr commented on src/app/auxiliary/auxiliary.component.css:
```diff
@@ -41,12 +41,19 @@ body {
+  top: 50%;
+  position: absolute;
+}
+
```
Floating Image class (in future: fix alignment, small border radius and etc.)

bxkr commented on src/app/auxiliary/auxiliary.component.html:
```diff
@@ -8,7 +12,7 @@
       <button mat-button class="class-elements" routerLink="/map">Карта</button>
       <div class="img-board">
-        <img  src='./favicon.ico' height=50px [matMenuTriggerFor]="logo_menu" alt="logo">
+        <img src='./favicon.ico' height=50px [matMenuTriggerFor]="logo_menu" alt="logo">
```
Remove excess whitespace

bxkr commented on src/app/pages/about/deviz/deviz.component.css:
```diff
@@ -0,0 +1,3 @@
+.line-division {
+  line-height: 2;
+}
```
Class of < pre > in about.component

bxkr commented on src/app/pages/contacts/contacts.component.css:
```diff
@@ -0,0 +1,3 @@
+#vk_groups {
+  margin-left: auto;
+}
```
VK Group widget core

bxkr commented on src/app/pages/gallery/gallery.component.css:
```diff
@@ -2,7 +2,9 @@
 }
+
 .hvrbox {
+  border-radius: 4px;
```
To make it beautiful

bxkr commented on src/index.html:
```diff
@@ -8,6 +8,10 @@
+  <script type="text/javascript" src="https://vk.com/js/api/openapi.js?168"></script>
+  <script type="text/javascript">
+    VK.Widgets.Group("vk_groups", {mode: 3}, 1214645);
+  </script>
```
VK Group widget import

bxkr commented on src/styles.css:
```diff
@@ -1,5 +1,27 @@
 /* You can add global styles to this file, and also import other style files */
+@import url('https://fonts.googleapis.com/css2?family=JetBrains+Mono&display=swap');
```
Font import

bxkr commented on src/styles.css:
```diff
@@ -1,5 +1,27 @@
+@import url('https://fonts.googleapis.com/css2?family=JetBrains+Mono&display=swap');
+
 * {
+  font-family: "JetBrains Mono", monospace;
```
Set font to all text

bxkr committed:
Fix white BG in some places

bxkr commented on src/app/auxiliary/auxiliary.component.css:
```diff
@@ -39,6 +39,7 @@
 body {
   font-family: Roboto, "Helvetica Neue", sans-serif;
   height: 100vh;
+  overflow-x: hidden;
```
Fix white BG in some places

bxkr committed:
Getting rid of VK scripts

bxkr committed:
Refactoring deviz-component content

bxkr committed:
Some code improvements

bxkr committed:
Logo floating menu

bxkr committed:
z-index improvement

bxkr committed:
Improving floating image performance
- Set flex display
- Center image
- Separate class for wide images

bxkr committed:
Splitting gallery items
Only for testing purposes
In the future, gallery items will be a grid

bxkr committed:
Floating image v1.0
- Reworking the appearance and disappearance of a floating image
- Soft transition for floating

bxkr commented on src/styles.css:
```diff
@@ -8,10 +8,13 @@
 #floating {
-  display: none;
-  background-color: rgba(0, 0, 0, 0.8);
+  display: flex;
```
For easy and adaptive alignment

bxkr commented on src/styles.css:
```diff
@@ -8,10 +8,13 @@
-  display: none;
-  background-color: rgba(0, 0, 0, 0.8);
+  display: flex;
+  background-color: rgba(0, 0, 0, 0);
```
For separate background opacity transition
